curriculum/schema/challengeSchema.js

```diff
@@ -91,6 +91,7 @@ const schema = Joi.object()
         crossDomain: Joi.bool()
       })
     ),
+    assignments: Joi.array(),
```

Comment:
```suggestion
    assignments: Joi.array().items(Joi.string()),
```
Might as well :man_shrugging: